pytorch_lightning/core/lightning.py

```diff
@@ -1908,6 +1910,8 @@ def to_torchscript(
         """
         mode = self.training
 
+        self._running_torchscript = True
```

Comment:
this should be reset at the end of the method

Reply:
Done!

Regarding the annotation, mypy does not require it.